ozaria/site/templates/play/level/tome/cast-button-view-ext.jade

```diff
@@ -0,0 +1,19 @@
+img.game-dev-play-btn(src="/images/ozaria/level/emerald.png", alt=$.i18n.t("common.play"))
```

Comment:
I think we need to convert the legacy cast button to the new run button instead of the game dev play button. 
Game dev play button is closer to the `Play` in the capstone level UX I think.